db/migrate/20191012043517_add_unconfirmed_email_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUnconfirmedEmailToUsers < ActiveRecord::Migration[5.0]
```

Comment:
We don't need to include this column as we are not using `reconfirmable`.  Please remove.